Update Readme (#3720)
ethereum/EIPs: README.md

Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards.

**Before you initiate a pull request**, please read the [EIP-1](https://eips.ethereum.org/EIPS/eip-1) process document. Ideas should be thoroughly discussed prior to opening a pull request on the [Ethereum Magicians forums](https://ethereum-magicians.org).

This repository tracks the ongoing status of EIPs. It contains:

> - [Draft](https://eips.ethereum.org/all#draft) proposals which indicates author is still making changes and is not ready for review.

Reviewer: ```suggestion
- [Draft](https://eips.ethereum.org/all#draft) proposals where the author(s) are still making changes and is not ready for review.
```

> - [Review ](https://eips.ethereum.org/all#review) for proposals ready for EIP review process and author is ready for improvement suggestions.

Reviewer: ```suggestion
- [Review ](https://eips.ethereum.org/all#review) for proposals ready for EIP review process and the author(s) are ready for improvement suggestions.
```
